src/components/TestButton.tsx

```diff
@@ -36,7 +36,7 @@ export function TestButton({ isDisabled, onTest }: Props) {
     <ControlButton
       aria-label={
         isDisabled
-          ? "Record a step in order to run a test"
+          ? "You cannot test your recorded tests until you have finished a recording session"
```

Comment:
In terms of copy, I think `test your recorded tests` sounds a bit weird. I'd probably use "_execute_ your recorded tests" or "_run_ your recorded tests".

Reply:
I'll push this update, well-spotted. Result of writing copy while focused on something else.